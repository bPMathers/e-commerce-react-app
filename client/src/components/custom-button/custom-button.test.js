import React from 'react';
import { shallow } from 'enzyme'

import CustomButton from './custom-button.component';

describe('CustomButton component', () => {
  let wrapper, mockProps;

  beforeEach(() => {
    mockProps = {
      children: 'mockChildren'
    }

    wrapper = shallow(<CustomButton {...mockProps} />);
  });

  it('should render CustomButton component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render its children', () => {
    expect(wrapper.find('CustomButtonContainer').text()).toBe('mockChildren');
  })
});