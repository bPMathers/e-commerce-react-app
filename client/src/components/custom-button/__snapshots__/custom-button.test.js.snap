// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton component should render CustomButton component correctly 1`] = `
<CustomButtonContainer>
  mockChildren
</CustomButtonContainer>
`;
